lib/commands/debug/helper.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path');
+const fs = require('fs');
+const { execSync } = require('child_process');
+const CONSTANTS = require('@src/utils/constants');
+
+module.exports = {
+    getHostedSkillInvocationInfo,
+    getNonHostedSkillInvocationInfo,
+    getNodeDebugAdapterPath,
+    getPythonDebugAdapterPath
+};
+
+const HOSTED_SKILL_INVOCATION_CONFIG = new Map([
+    [
+        'node', {
+            skillCodeFolderName: 'lambda',
+            handlerName: 'handler',
+            skillFileName: 'index'
+        }
+    ],
+    [
+        'python', {
+            skillCodeFolderName: 'lambda',
+            handlerName: 'lambda_handler',
+            skillFileName: 'lambda_function'
+        }
+    ]
+]);
+
+function getHostedSkillInvocationInfo(runtime) {
+    for (const [key, value] of HOSTED_SKILL_INVOCATION_CONFIG.entries()) {
+        if (runtime.includes(key)) {
+            return value;
+        }
+    }
+}
+
+function getNonHostedSkillInvocationInfo(runtime, handler, skillCodeFolderName) {
+    if (runtime.includes('java')) {
+        return { handlerName: handler, skillCodeFolderName };
+    }
+    const handlerInfo = handler.split('.');
+    const handlerName = handlerInfo.pop();
+    const skillFileName = handlerInfo.join(path.delimiter);
+    return { handlerName, skillFileName, skillCodeFolderName };
+}
+
+function getNodeDebugAdapterPath(skillFolderPath) {
+    const script = path.join(skillFolderPath, CONSTANTS.LOCAL_DEBUG.NODE.SCRIPT_LOCATION);
+    if (!fs.existsSync(script)) {
+        throw ('The module ask-sdk-local-debug is missing. '
```

Comment:
Maybe this: "ask-sdk-local-debug cannot be found. Please install ask-sdk-local-debug to your skill project."